src/es/components/molecules/tabs/Tabs.js

```diff
@@ -39,17 +39,17 @@ export default class Tabs extends Shadow() {
             font-family: var(--font-family);
         }
         :host .tab-navigation {
-            border-bottom: 1px solid var(--m-gray-300);
+            border-bottom: var(--border-bottom, 1px solid var(--m-gray-300));
             list-style: none;
             display: flex;
             padding: 0;
             gap: 1.5em;
-            margin-bottom: 3em;
+            margin-bottom: var(--tab-navigation-margin-bottom, 3em);
```

Comment:
try to use the same selectors within a css statement. ether use always --tab-navigation or avoid it:

--tab-navigation-margin-bottom vs --margin-bottom